lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1838,6 +1838,82 @@ class ConvertAtenGatherOp : public OpConversionPattern<AtenGatherOp> {
 };
 } // namespace
 
+namespace {
+class ConvertAtenBroadcastToOp
+    : public OpConversionPattern<AtenBroadcastToOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(AtenBroadcastToOp op, llvm::ArrayRef<Value> operands, ConversionPatternRewriter &rewriter) const override {
+
+    // Get variables
+    if (failed(verifyLinalgCompatibleTypes(op, rewriter)))
+      return failure();
+    AtenBroadcastToOp::Adaptor adaptor(operands);
+    Value inTensor = adaptor.self();
+    auto inType = inTensor.getType().cast<RankedTensorType>();
+    auto origShape = inType.getShape();
+    auto elementType = inType.getElementType();
+    auto loc = op.getLoc();
+
+    SmallVector<int64_t> inShape, outShape;
+    SmallVector<AffineExpr> outExpr;
+    if (!matchPattern(adaptor.size(), m_TorchConstantIntList(inShape)))
+      return rewriter.notifyMatchFailure(op, "dimension list must be constant");
```

Comment:
m_TorchConstantIntList restricts us to the case that all dimensions are constants, which we don't want (that would restrict us to static sizes, which we don't want).

It's ok if there are dynamic values in the list. It just needs to be defined by PrimListConstructOp

Reply:
could use `getListConstructElements` instead.